.github/workflows/emscripten.yml

```
@@ -0,0 +1,144 @@
+# Copied from NumPy https://github.com/numpy/numpy/pull/25894
+# https://github.com/numpy/numpy/blob/d2d2c25fa81b47810f5cbd85ea6485eb3a3ffec3/.github/workflows/emscripten.yml
+#
+
+name: Test Emscripten/Pyodide build
```

```suggestion
name: Pyodide CI
```

based on discussions in https://github.com/sympy/sympy/issues/24920. Should I make the job name shorter below?

---

Addressed in 0d81b906191caacb87deb7dc614d036d3b78f805.